### ISSUE TITLE
When you try to buy a plot by clicking on the button, it does not work. The "space" key - it turns out.

### ISSUE THREAD

**Issue by vitalii2011** [Mmtrx/FS22_BetterContracts#88]
```
Error: Running LUA method 'mouseEvent'.
C:/Users/…/Documents/My Games/FarmingSimulator2022/mods/FS22_BetterContracts/scripts/options.lua:379: attempt to call local 'superf' (a table value)
```

**Comment by Mmtrx** [comment]
Sorry, closed inadvertently.
Here is a fix for that: [v1.2.6.4](https://github.com/Mmtrx/FS22_BetterContracts/releases/tag/v1.6.2.4)

**Comment by vitalii2011** [comment]
In v1.2.6.4 this problem is gone. Thank you.
